README.md

```diff
@@ -45,6 +45,37 @@ Or if you want paid support please contact Catalyst IT Australia:
 https://www.catalyst-au.net/contact-us
 
 
+3. Set up master password feature for load test (Non Production Environments)
+
+Add this setting to config.php:
+
+```
```

Comment:
where possibly it's nice to declare the language for markdown code blocks add php after the opening triple quote

```
$CFG->forced_plugin_settings = array(
    'auth_basic' => array(
        'master' => 'masterpassword',
    ),
);
```
vs

```php
$CFG->forced_plugin_settings = array(
    'auth_basic' => array(
        'master' => 'masterpassword',
    ),
);
```